### PR TITLE
Read range from config.json and added checks as needed

### PR DESCRIPTION
**Pull request by ramesh-jr**

### CODE REVIEW

**Comment by vinitshahdeo** [comment]
@ramesh-jr Thanks for raising the PR. Will look into this soon.

> @Kashish121 Please review this.

File: scanner.py [hunk 1]
```diff
@@ -19,10 +20,22 @@
...
+        config = json.load(config_file)
+    range_high = int(config['range']['high'])
+    range_low = int(config['range']['low'])
+    
+except FileNotFoundError:
+    print("config.json file not found")
+except ValueError:
+    print("Kindly check the json file for appropriateness of range")
+
 # scanning the port only in range of (1, 8888)
```
Comment (Kashish121) [inline scanner.py:35]: Can be changed to,
#scanning the port only in range of (range_low,range_high):

**Review by Kashish121 — APPROVED** [review]
Left a comment. Approved changes.

File: scanner.py [hunk 2]
```diff
@@ -19,10 +20,22 @@
...
 
 try:
-    for port in range(1,8888):  
+    for port in range(range_low,range_high):
```
Comment (vinitshahdeo) [inline scanner.py:38]: @ramesh-jr Linting issue here.

File: scanner.py [hunk 3]
```diff
@@ -19,10 +20,22 @@
...
+try:
+    with open('config.json') as config_file:
+        config = json.load(config_file)
+    range_high = int(config['range']['high'])
+    range_low = int(config['range']['low'])
+    
+except FileNotFoundError:
+    print("config.json file not found")
+except ValueError:
+    print("Kindly check the json file for appropriateness of range")
```
Comment (vinitshahdeo) [inline scanner.py:33]: @ramesh-jr Better to mention `config.json` in place of **json file**.

**Review by vinitshahdeo — CHANGES_REQUESTED** [review]

File: scanner.py (hunk 2 above)
Comment (ramesh-jr) [reply scanner.py]: @vinitshahdeo Hi,Vinit. I couldn't see any linting issue on this line. Could you kindly elaborate?

**Pull request closed without merging**